transform/slices_test.go

```diff
@@ -0,0 +1,56 @@
+package transform
```

Comment:
`stringInSlice` is not being exported in that package, it it must be `transform`